package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "build": "yarn run clean && yarn run prep && NODE_ENV=production webpack --progress --colors",
     "clean": "rm -rf build/",
-    "chimp-test": "chimp chimp.config.js",
+    "chimp-test": "chimp chimp.config.js --timeout 10000",
```

Comment:
Can you instead add this to `chimp.config.js`? I'd like to keep all of our config there instead of using the cli flags; just easier to view/edit later

Reply:
Yes, I will, also because chimp does not pick this up :-)